weed/util/http/volume/http_util.go

```diff
@@ -0,0 +1,72 @@
+package volume
```

Comment:
try to use different package name other than filer, master, volume, http. They are already used or too general.

also, maybe change http_unknown => http_general

Reply:
Chris, hello!

> try to use different package name other than filer, master, volume, http. They are already used or too general.
Got it, thanks!

> also, maybe change http_unknown => http_general
I temporarily added `http_unknown`. I was thinking of fixing `http_unknown` to `volume`/`filter`/`master` http clients (if they are used in the code) in the next stage (next commits to this branch).

Can you tell me if there is a way to determine which client functions from `weed/util/http_util.go` were used for?
And what should I pay attention to? For example, are there functions in the code that can simultaneously use `volume`/`filer`/`master` clients (besides `weed/util/http_util.go`)? If there are functions that are used by different clients, then perhaps my solution will be incorrect, and it will be necessary to add client initialization to the config objects of the microservices during initialization (and their transfer to subsequent functions).

Reply:
Not possible to tell. Maybe add a parameter during execution time.